typescript/sdk/src/deploy/HyperlaneDeployer.ts

```diff
@@ -399,10 +400,21 @@ export abstract class HyperlaneDeployer<
     );
 
     if (initializeArgs) {
-      this.logger.debug(`Initialize ${contractName} on ${chain}`);
-      const overrides = this.multiProvider.getTransactionOverrides(chain);
-      const initTx = await contract.initialize(...initializeArgs, overrides);
-      await this.multiProvider.handleTx(chain, initTx);
+      if (
+        await isInitialized(
+          this.multiProvider.getProvider(chain),
+          contract.address,
+        )
```

Comment:
nit: semicolon

Reply:
This is the if condition, so i dont think a semicolon is applicable here